.travis.yml

```diff
@@ -34,6 +52,18 @@ install:
   - if [[ $TRAVIS_PYTHON_VERSION != 'pypy'* ]]; then
         pip install numpy scipy;
     fi
+  - if [[ $TRAVIS_EVENT_TYPE == "cron" ]]; then
```

Comment:
Instead of testing on `$TRAVIS_EVENT_TYPE` can you please test on `[[ "$PROJECT" != "" ]]`